docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -75,67 +67,35 @@ WARNING: Listening on any other interface than the local loopback poses a securi
 
 * `path`: The URL of the endpoint.
 
-* `unsecuredPaths`: A comma-separated list of endpoint paths that are to stay unsecured even if the `secure` attribute is set to `true`.
-
-* `cookiesAuthEnabled`: When set to `true` (the default is `false`), the JWT workspace token is automatically fetched and included in a workspace-specific cookie to allow requests to pass through the JWT proxy.
-+
-WARNING: This setting potentially allows a link:https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSRF] attack when used in conjunction with a server using POST requests.
+When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as an `exposure` port. This is useful for debugging a web application, for example. It is impossible to do this for servers that autostart with the container (for example, a database server). For such components, specify the endpoints explicitly.
```

Comment:
IBM SG: Parentheses in General Text (p.97): Avoid using parentheses in general text because text in parentheses is decreased in importance. Include information that is important and eliminate information that is not. Parentheses can also interrupt the flow of a sentence. Try to rewrite the text or use other punctuation instead of parentheses. If you must write a sentence that includes text in parentheses, ensure that the sentence is grammatically correct and technically accurate if you omit the parenthetical text.